pkg/controller/provider/container/vsphere/model.go

```diff
@@ -397,7 +397,7 @@ func (v *NetworkAdapter) Apply(u types.ObjectUpdate) {
 		switch p.Op {
 		case Assign:
 			switch p.Name {
-			case fTag:
+			case fTag, fKey:
```

Comment:
Did you mean to assign to the key to the tag?

Reply:
Nope, I certainly did not.